Gems/AWSGameLift/cdk/aws_gamelift/flex_match/matchmaking.py

```diff
@@ -0,0 +1,60 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import typing
+
+from aws_cdk import (
+    core,
+    aws_gamelift as gamelift
+)
+
+from . import flex_match_configurations
+
+FLEX_MATCH_MODE = 'WITH_QUEUE'
+
+
+class Matchmaking:

```

Comment:
nit: This class is not doing matching its MatchmakingResources or MatchmakingSetup or similar.